src/SwipeProfile.css

```diff
@@ -1,5 +1,107 @@
 .SwipeProfile {
-    text-align: center;
-    margin: auto;
-    padding: 20px;
-  }
\ No newline at end of file
+  text-align: center;
+  margin: auto;
+  position: relative;
+}
+ul {
+  /*unordered list for navigation bar*/
+  background-color: #f8dabb;
+  list-style-type: none;
+  overflow: hidden;
+  margin: 0;
+  padding: 0;
+}
+
+li {
+  float: left; /*to get navigation bar items to slide next to each other*/
+}
+
+li a {
+  text-align: center;
+  padding: 14px 16px;
+  display: block;
+  color: black;
+  text-decoration: none;
+}
+
+li a:hover:not(.active) {
+  background-color: rgb(
+    102,
+    102,
+    102
+  ); /*color change when mouse hovers over the list item on navigation bar*/
+}
+
+.active {
+  background-color: #f7dfc7;
+}
+
+.profileIntroSection {
+  /*turquoise section that includes other info in smaller font*/
+  background-color: #4dadb7;
+  align-items: center;
+  margin-right: auto;
+  margin-left: auto;
+  width: 500px;
+  border-radius: 12px;
+}
+
+.letsTalkButton {
+  /*"yes, let's match" button */
+  background-color: #e96a6a;
+  border: 2px solid black;
+  width: 240px;
+  color: white;
+  padding: 10px 35px;
+  text-align: center;
+  font-size: 20px;
+  position: relative;
+  margin: 1px 1px;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+.notInterestedButton {
+  /*"no, reject match" button*/
+  background-color: #3680a3;
+  border: 2px solid black;
+  width: 240px;
+  color: white;
+  padding: 10px 35px;
+  text-align: center;
+  font-size: 20px;
+  position: relative;
+  margin: 1px 1px;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+.pageTitle {
+  font-size: 30px;
+}
+
+.letsTalkButton:hover {
+  /*when user hovers over this button, color of button changes*/
+  background-color: #e25959;
+  color: white;
+  border: 3px solid;
+}
+
+.letsTalkButton:focus {
+  /*when user selects this button, color of button changes*/
+  background-color: #e96a6a;
+  color: white;
+  border: 3px solid;
+}
+
```

Comment:
whats the difference between focus and hover? the page changes when the user selects a button so is the focus needed? (this is the same for the not interested button

Reply:
Yeah hover allows the button to change color when your mouse hovers over the button. Focus allows the button to change to a diff color once it has been selected. Both are good visual cues for the user to let her/him know that s/he has done an action